_data/health.yml

```diff
@@ -27,21 +29,24 @@ websites:
       doc: https://www.drchrono.com/blog/two-factor-authentication-extra-security-for-your-health-records/
 
     - name: Drugs.com
-      url: http://www.drugs.com
+      url: https://www.drugs.com/
       twitter: DrugsCom
+      facebook: Drugscom
       img: drugscom.png
       tfa: No
       status: https://twitter.com/Drugscom/status/493856207572975620
 
     - name: Elation Health
       url: https://www.elationhealth.com/
       twitter: ElationHealth
+      facebook: ElationHealth
```

Comment:
With only 189 likes and by the looks of it completely unmonitored by the company it might be better not to list the company at all. They also only have 573 followers on Twitter with no replies.